app/generate_command_metadata.py

```diff
@@ -120,9 +223,15 @@ def generate_command_info(command_definition_files, template_file):
     def get_command(line):
         return line.split(' ')[1]
 
+    def get_command_l10n(line):
+        command = get_command(line)
+        if command in EXISTING_TRANSLATIONS:
+            return EXISTING_TRANSLATIONS[command]
+        return 'IDS_' + command
```

Comment:
If someone adds a new command but they don't add a string in the grdp for it (or mark it in EXISTING_TRANSLATIONS, does the build fail? Which I suppose is a good thing if so.